### PR TITLE
Aggregator: add failed_jobs metrics

### PR DESCRIPTION
**Pull request by wkq7740**

### CODE REVIEW

File: nostr-client-plus/src/bin/aggregator.rs [hunk 1]
```diff
@@ -171,11 +174,13 @@ async fn run() -> Result<()> {
                                 match validate_result {
                                     Err(msg) => {
                                         tracing::error!("Validate error {}", msg.to_string());
+                                        FAILED_JOBS.inc();
```
Comment (dongshu2013) [inline nostr-client-plus/src/bin/aggregator.rs:177]: can we separate these two? so we can have per error reason counters

**Comment by dongshu2013** [comment]
for assignments not finished by any workers, do we have a way to track them? Now per the dashboard, we assigned ~45K jobs but only ~13K of them are finished. Wondering how we can track these jobs

File: nostr-client-plus/src/bin/aggregator.rs (hunk 1 above)
Comment (wkq7740) [reply nostr-client-plus/src/bin/aggregator.rs:177]: I dont see the necessary, there are actually about 5 fail cases in the verify result, seperate into 5 is too over complicated

**Comment by wkq7740** [comment]
> for assignments not finished by any workers, do we have a way to track them? Now per the dashboard, we assigned ~45K jobs but only ~13K of them are finished. Wondering how we can track these jobs



> for assignments not finished by any workers, do we have a way to track them? Now per the dashboard, we assigned ~45K jobs but only ~13K of them are finished. Wondering how we can track these jobs

dont know yet

**Comment by dongshu2013** [comment]
> > for assignments not finished by any workers, do we have a way to track them? Now per the dashboard, we assigned ~45K jobs but only ~13K of them are finished. Wondering how we can track these jobs
> 
> > for assignments not finished by any workers, do we have a way to track them? Now per the dashboard, we assigned ~45K jobs but only ~13K of them are finished. Wondering how we can track these jobs
> 
> dont know yet

let's create a ticket and track this issue, we need a design for this eventually

File: nostr-client-plus/src/bin/aggregator.rs (hunk 1 above)
Comment (dongshu2013) [reply nostr-client-plus/src/bin/aggregator.rs:177]: @wkq7740 I merged the diff but let's separate these two so we have better visibility

**Pull request merged**